http/lang/en/about.lang

```diff
@@ -2,7 +2,7 @@ about			About
 
 contact			Contact us
 
-contact_text			You can contact us by email at %s, by private message on the game, by Facebook or Twitter.
+contact_text			You can contact us by email at %s or by private message on the game, and by Facebook or Twitter.
```

Comment:
I think this one is getting even more awkward. It is valid, but just listing all the possibilities without grouping them seems more appropriate. Eg. `You can contact us via email at %s, private message on the game, Facebook, or Twitter.` That's just my opinion though.